apps/frontend/src/app/Components/Character/CharacterCardPico.tsx

```diff
@@ -16,13 +20,29 @@ import CharacterCard from './CharacterCard';
 
 export default function CharacterCardPico({ characterKey = "", index = -1, onClick, simpleTooltip = false, }:

```

Comment:
I think in this case, this might be over-nested. I think `CharacterCardPico` should have a strict `character:CharacterKey` prop, and you alternatively export a separate `EmptyCharCardPico` with `index`. This prevents prop drilling, and enforces typing.